modules/memberships/class-jetpack-memberships.php

```diff
@@ -276,5 +286,44 @@ public static function get_blog_id() {
 	public static function get_connected_account_id() {
 		return get_option( self::$connected_account_id_option_name );
 	}
+
+	/**
+	 * Whether Memberships (aka Recurring Payments) are enabled.
```

Comment:
Nit; this could just be _"Whether Recurring Payments are enabled."_, no?

Reply:
```suggestion
	 * Whether Recurring Payments are enabled.
```